### PR TITLE
Report/license

### PR DESCRIPTION
**Pull request by AugustBugge**

### CODE REVIEW

File: docs/report.md [hunk 1]
```diff
@@ -68,12 +68,20 @@ Briefly describe what kinds of tests you have in your test suites and what they
 
 ### License
```
Comment (Oliverst8) [inline docs/report.md:70]: Please add a beginning to the section, since it starts very sudden

File: docs/report.md [hunk 2]
```diff
@@ -68,12 +68,20 @@ Briefly describe what kinds of tests you have in your test suites and what they
...
 
-State which software license you chose for your application.
+None of the libraries used has the GPL license, or any other copyleft license. 
+Therefore the choice of license was open, and the MIT license was used.
```
Comment (Oliverst8) [inline docs/report.md:72]: Therefore the choice of license was open -> Therefore the choice of license was _left_ open

Maybe?

File: docs/report.md [hunk 3]
```diff
@@ -68,12 +68,20 @@ Briefly describe what kinds of tests you have in your test suites and what they
 
 ### License
 
-State which software license you chose for your application.
+None of the libraries used has the GPL license, or any other copyleft license. 
+Therefore the choice of license was open, and the MIT license was used. 
+The MIT license is one of the most permissive licenses, allowing the program to be used for almost anything as long as the original copyright notice and license is included. 
+It also means the program can be used as is, and the developers have no responsibility for maintaining the product.
```
Comment (Oliverst8) [inline docs/report.md:74]: Maybe add something about we we want this

File: docs/report.md (hunk 2 above)
Comment (Oliverst8) [reply docs/report.md:72]: And the MIT license was chosen

File: docs/report.md [hunk 4]
```diff
@@ -68,12 +68,20 @@ Briefly describe what kinds of tests you have in your test suites and what they
 
 ### License
 
-State which software license you chose for your application.
+None of the libraries used has the GPL license, or any other copyleft license. 
+Therefore the choice of license was open, and the MIT license was used. 
+The MIT license is one of the most permissive licenses, allowing the program to be used for almost anything as long as the original copyright notice and license is included.
```
Comment (Oliverst8) [inline docs/report.md:73]: "license _is_ included" -> "license _are_ included"

File: docs/report.md (hunk 3 above)
Comment (Oliverst8) [inline docs/report.md:74]: Maybe change the beginning _It_ to _this_

**Review by Oliverst8 — CHANGES_REQUESTED** [review]

**Review by JonathanVil — APPROVED** [review]
Looks good

**Review by Oliverst8 — APPROVED** [review]

**Pull request merged**